debian/Dockerfile

```diff
@@ -6,7 +6,7 @@ ARG METAL_NETWORKER_VERSION
 FROM ghcr.io/metal-stack/frr:${FRR_VERSION} AS frr-artifacts
 FROM ghcr.io/metal-stack/metal-networker:${METAL_NETWORKER_VERSION} as metal-networker
 
-FROM mirror.gcr.io/library/golang:1.15 as ignition-builder
+FROM mirror.gcr.io/library/golang:1.14 as ignition-builder
```

Comment:
I think it is better to use golang:1.15 like i did here: https://github.com/metal-stack/metal-images/pull/85/files#diff-b98298742c5536fa90710aea60f865cdf0149b61c06288c4a0219ce4796ec807R9